avm-transpiler/src/instructions.rs

```diff
@@ -18,7 +18,7 @@ pub struct AvmInstruction {
     /// The 0th bit corresponds to an instruction's 0th offset arg, 1st to 1st, etc...
     pub indirect: Option<AvmOperand>,
 
-    /// Some instructions have a destination xor input tag
+    /// Some instructions have a destination or input tag
```

Comment:
@fcarreiro is this correction correct?

Reply:
Using ‘or’ is more appropriate here, as it indicates that the instruction can have either a destination tag or an input tag, but not necessarily both.

Reply:
the 'not necessarily both' part is what 'xor' would mean, though :)

Reply:
You're right! 'XOR' captures the idea that only one of the tags can be present, but using 'or' might make it clearer to those who aren't familiar with the terminology. Maybe we could stick with 'or' for simplicity?

Reply:
The tag concept changed since this was written. Right now only 2 opcodes have (and must have) a tag: SET and CAST. In both cases, the tag is the expected destination type to be set. All other opcodes don't have (and cannot have) a tag.

I'm ok with "Some instructions have a tag, it's usage will depend on the instruction." at this level. I'm also ok with removing "xor" since it's not real english.